### ISSUE TITLE
Icons needed for extremely popular shops [meta-issue]

### ISSUE THREAD

**Issue by kocio-pl** [gravitystorm/openstreetmap-carto#1460]
We should start rendering extremely popular shop types (let's say above 10 000 uses [on taginfo](http://taginfo.openstreetmap.org/keys/shop#values)), but we probably have no icons for them:

* [x] 1. electronics (19 671) -  see: https://github.com/gravitystorm/openstreetmap-carto/pull/1493
* [x] 2. hardware (17 504) - see: https://github.com/gravitystorm/openstreetmap-carto/pull/1522
* [x] 3. gift (14 457) - see: https://github.com/gravitystorm/openstreetmap-carto/pull/1493
* [x] 4. beauty (14 258) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1525
* [x] 5. chemist (13 281) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1528
* [x] 6. garden_centre (11 509) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1523
* [x] 7. sports (10 554) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1530
* [x] 8. variety_store (10 249) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1536
* [x] 9. confectionery (10 091) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1534
* [x] 10. car_parts (10 003) - see https://github.com/gravitystorm/openstreetmap-carto/issues/1537
* [x] 11. travel_agency (10 120) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1409
* [x] 12. laundry + dry_cleaning (9 893 + 7 786) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1411
* [x] 13. newsagent + kiosk (8 913 + 46 299) - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1459
* [x] 14. stationery - see: https://github.com/gravitystorm/openstreetmap-carto/pull/1588
* [x] 15. beverages  - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1592
* [x] 16. toys - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1414

Getting there (items above 5 000 uses):
* [x] amenity/shop=ice_cream - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1479
* [x] outdoor - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1717
* [x] copy shop (see https://github.com/gravitystorm/openstreetmap-carto/issues/1413)
* [ ] boutique + fashion - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1706
* [ ] deli (jar?)
* [ ] tyres (bieżnik?)

Others (some useful items above 1 000 uses):
* [x] musical instruments shop (see https://github.com/gravitystorm/openstreetmap-carto/issues/1420)
* [x] bag - see: https://github.com/gravitystorm/openstreetmap-carto/pull/1801
* [ ] tobacco + e-cigarettes - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1618
* [ ] interior_decoration + houseware - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1406
* [ ] charity - see: https://github.com/gravitystorm/openstreetmap-carto/issues/1869
* [ ] ticket
* [ ] fabric (kokarda?)
* [ ] art + paint (palette?)
* [ ] funeral_directors (coffin?)
* [ ] locksmith (key)

Some other (including super-star shop=kiosk), which already have the icons available, were proposed in https://github.com/gravitystorm/openstreetmap-carto/issues/1402.

**Comment by nebulon42** [comment]
We have for gift: [gift-18.svg](https://github.com/nebulon42/osmic/blob/master/shop/gift-18.svg): ![gift-18](https://cloud.githubusercontent.com/assets/3531092/6779973/4625f850-d161-11e4-8793-27c4368c9ba5.png)

Regarding `shop=hardware` the distinction to `shop=doityourself` is not clearly defined, so I think for a start we could also use the DIY shop icon for that.

**Comment by kocio-pl** [comment]
OK, the gift icon is good and +1 for sharing the doityourself icon.

**Comment by nebulon42** [comment]
Regarding `shop=electronics` I'm not sure (see also https://github.com/nebulon42/osmic/issues/17) which icon to choose. There are subtypes like computer, hifi, photo, etc. so none of those is suitable as icon. Ideas?

**Comment by kocio-pl** [comment]
If we avoid popular subtypes, which may be needed later, we would be left with some generic/iconic electronics like TV-set or a small radio with antenna. Taginfo may help a bit: according to http://taginfo.openstreetmap.org/keys/electronics_repair radio should be the safest choice, since it's not even mentioned and it's probably the oldest (=the least interesting) kind of electronics still used today (phone is rather mobile phone these days).

**Comment by nebulon42** [comment]
Yes, TV and/or radio could actually work.

**Comment by nebulon42** [comment]
@kocio-pl: Could you please update your list and cross out things that have been done already? This would make it easier to see what is still open.

**Comment by math1985** [comment]
I added checkboxes. 1 and 3 should be checked, right?

I'm not sure if regular users can check/uncheck checkboxes?

**Comment by nebulon42** [comment]
Thanks, yes that's fine. I can't check them here, but for my own comments as in #1165 I can.

**Comment by HolgerJeromin** [comment]
Changing checkboxes is a normal edit in the background of github. So only the owner and repo maintainer is able to change the content.

**Comment by nebulon42** [comment]
@kocio-pl `shop=hardware` doesn't currently have an icon, so this should be unchecked.

**Comment by kocio-pl** [comment]
Wasn't it [your idea](https://github.com/gravitystorm/openstreetmap-carto/issues/1460#issuecomment-84974146) to use [DIY icon](https://github.com/gravitystorm/openstreetmap-carto/pull/1380)? =}

**Comment by nebulon42** [comment]
Idea yes, but it's not done yet. When checked we might think it is done while it isn't. I will make that PR after #1520 got decided, otherwise there will be certainly merge conflicts in one of them.

**Comment by kocio-pl** [comment]
OK, I see. Unchecked.

**Comment by SomeoneElseOSM** [comment]
Just to provide a bit of external feedback, I noticed this on the forum:

http://forum.openstreetmap.org/viewtopic.php?pid=504204#p504204

I suspect that the effect that they're talking about is as seen here at z17:

http://www.openstreetmap.org/#map=17/52.95354/-1.14638

(I'm not convinced that it's a "problem" per se; just trying to link to some feedback).

**Comment by nebulon42** [comment]
Probably we should start discussing if we should render some POIs later on. For shops we could also switch from a two-tier (supermarkets etc. earlier and everything later) to a three-tier rendering. It may be good to have a separate issue for that.

**Comment by kocio-pl** [comment]
I think it really is a problem, but much more general: we have a threshold for rendering some POIs (or other elements), but since we have no objects density metric available, some places will be overloaded, while at the same time other ones will be almost blank with the same style. 

There was similar discussion about footways: in the city, in cemetery or in allotments red dotted lines can be quite disturbing on some zoom levels, but in remote places (in the forest, in the mountains and the like) they can be even the most important thing for the users to see.

To some degree we could deal with it better if we render some regions (countries, cities...) in a different way, but developers choice is that style should be uniform for the whole planet.

**Comment by nebulon42** [comment]
I still opened #1560 for that.

**Comment by kocio-pl** [comment]
Currently I don't see the possibility to add much more icons to this style and all the most important ones (10k+) are now rendered, so I close this issue. However it can still play the role of cheat sheet for adding some other interesting/popular shop types in the future.

**Comment by math1985** [comment]
Thanks for all your work!

**Comment by kocio-pl** [comment]
Interesting icon was [proposed](Map Icons/Proposed Icons) for shop=deli, what do you think about it?

http://wiki.openstreetmap.org/wiki/File:Shop_deli.svg

**Comment by dieterdreist** [comment]
sent from a phone

> Il giorno 19 giu 2016, alle ore 15:52, kocio-pl <notifications@github.com> ha scritto:
> 
> what do you think about it?


I can understand it, if I am told what it's meant to be, but otherwise I'd see it more for shop=seafood

**Comment by daganzdaanda** [comment]
Used at the size of the map icons, I believe that it won't be readable.

**Comment by kocio-pl** [comment]
Look at the "proposed" link - it looks quite good there.

**Comment by Jotam** [comment]
I'm the one who submitted the proposition for shop=deli. I chose a lobster because I believe it to be a generally recognized synonym for luxurious food.

**Comment by Jotam** [comment]
Can I add in this one for shop=dairy? https://wiki.openstreetmap.org/wiki/File:Shop_dairy-32x32.svg
What do you think?

**Comment by kocio-pl** [comment]
Just the remark: there is no map layer called "Mapnik", because it's just a name of the rendering software which is used with different styles. This one is called "osm-carto" in short and it would be good to fix the naming on the wiki one day. 

So "we" not "them"... ;-) But you're rightt, we use fish symbol here since my proposition (https://github.com/gravitystorm/openstreetmap-carto/pull/1617) has been merged.

**Comment by kocio-pl** [comment]
@Jotam What about something like [jar](https://github.com/openstreetmap/josm/blob/mirror/images_nodist/presets/deli.png)?

**Comment by Jotam** [comment]
In my opinion, the jar looks like a sweet shop. I never quite understood why it was used for shop=deli. 

When I think about it, the lobster is still the first thing that comes to my mind when talking about luxurious food.

I see why one might say it could stand for seafood. But then, **any** icon you could possibly think of for a deli could stand for something else. Like the jar for sweets, a ham for a butcher, a cheese for a cheese shop, or a bottle of champagne for a beverage shop. (Also see example pic on the wiki for a typical deli: https://wiki.openstreetmap.org/wiki/File:Rome_Italian_deli.jpg)

@kocio-pl Could you try to render the lobster and share the results here? Thanks a lot!

**Comment by kocio-pl** [comment]
For me jar would be good, especially with the visible hood, which for me is a symbol of exclusive, hand crafted item (it hints "deli" articles in convenience shops I visit):

http://pl.depositphotos.com/5893158/stock-illustration-jar-of-jam.html

Sure, I'll make this rendering and will post it here.
